tests/codex/stores/testrepostore.nim

```diff
@@ -262,56 +243,45 @@ asyncchecksuite "RepoStore":
 
   test "Should update block expiration timestamp when new expiration is farther":
     let
-      duration = 10
       blk = createTestBlock(100)
-      expectedExpiration: SecondsSince1970 = now + duration
-      updatedExpectedExpiration: SecondsSince1970 = expectedExpiration + 10
-      expectedKey = Key.init((BlocksTtlKey / $blk.cid).tryGet).tryGet
+      expectedExpiration = BlockExpiration(cid: blk.cid, expiry: now + 10)
+      updatedExpectedExpiration = BlockExpiration(cid: blk.cid, expiry: now + 20)
 
-    (await repo.putBlock(blk, some duration.seconds)).tryGet
+    (await repo.putBlock(blk, some 10.seconds)).tryGet
 
-    var response = await queryMetaDs(expectedKey)
+    let expirations = await getExpirations()
 
     check:
-      response.len == 1
-      !response[0].key == expectedKey
-      response[0].data == expectedExpiration.toBytes
+      expectedExpiration in expirations
 
-    (await repo.ensureExpiry(blk.cid, updatedExpectedExpiration)).tryGet
+    (await repo.ensureExpiry(blk.cid, now + 20)).tryGet
 
-    response = await queryMetaDs(expectedKey)
+    let updatedExpirations = await getExpirations()
 
     check:
-      response.len == 1
-      !response[0].key == expectedKey
-      response[0].data == updatedExpectedExpiration.toBytes
+      expectedExpiration notin updatedExpirations
+      updatedExpectedExpiration in updatedExpirations
 
-  test "Should not update block expiration timestamp when current expiration is farther then new one":
+  test "Should update block expiration timestamp when new expiration is farther":
```

Comment:
This one does the opposite, which is correct. The title is a copy-paste error.

Reply:
Ah, you're right, thanks for catching it!